packages/plugin-sharp/src/functions.ts

```diff
@@ -0,0 +1,57 @@
+import sharp from "sharp";
+
+type optNum = number | undefined;
+
+function fmtImage(converter: sharp.Sharp, format?: keyof sharp.FormatEnum) {
+    if (format) converter.toFormat(format);
+    return converter.toBuffer();
+}
+
+export function loadImage(image: string | Buffer, format?: keyof sharp.FormatEnum) {
+    const converter = sharp(image);
+    if (format) converter.toFormat(format);
+    return converter.toBuffer();
+}
+
+export function resizeImage(image: string | Buffer, width: number, height: number, format?: keyof sharp.FormatEnum) {
+    return fmtImage(sharp(image).resize(width, height), format);
+}
+
+export function rotateImage(image: string | Buffer, angle: number, format?: keyof sharp.FormatEnum) {
+    return fmtImage(sharp(image).rotate(angle), format);
+}
+
+export function blurImage(image: string | Buffer, intensity: optNum = undefined, format?: keyof sharp.FormatEnum) {
+    return fmtImage(sharp(image).blur(intensity), format);
+}
+
+export function compositeImage(image: string | Buffer, options: sharp.OverlayOptions[], format?: keyof sharp.FormatEnum) {
+    return fmtImage(sharp(image).composite(options), format);
+}
+
+export function flattenImage(image: string | Buffer, flatten: boolean | undefined | sharp.FlattenOptions = undefined, format?: keyof sharp.FormatEnum) {
+    return fmtImage(sharp(image).flatten(flatten), format);
+}
+
+export function flipImage(image: string | Buffer, format?: keyof sharp.FormatEnum) {
+    return fmtImage(sharp(image).flip(), format);
+}
+
+export function flopImage(image: string | Buffer, format?: keyof sharp.FormatEnum) {
```

Comment:
what's the difference between this and `flipImage`

Reply:
flip is horozontal flip and flop is vertical flip according to sharp documentation.

Reply:
@scientific-dev 
![image](https://user-images.githubusercontent.com/46562212/143682915-0401a9a3-a23a-484c-be57-4f15905e8b0f.png)

Reply:
Ahhh I made a mistake over there...